vnext/Microsoft.ReactNative/XamlUIService.idl

```diff
@@ -25,6 +25,8 @@ namespace Microsoft.ReactNative {
     // This needs to be manually provided to the ReactInstanceSettings when using XamlIslands
     static void SetXamlRoot(IReactPropertyBag properties, XAML_NAMESPACE.XamlRoot xamlRoot);
     static XAML_NAMESPACE.XamlRoot GetXamlRoot(IReactPropertyBag properties);
-  }
 
+    static UInt64 GetIslandWindow(IReactPropertyBag properties);
```

Comment:
>GetIslandWindow [](start = 18, length = 15)

Should we add the "Handle" suffix to these new method names for clarity? To differentiate it from storing actual XAML Window control.

Reply:
I'm torn on this. I would like to use the WindowId type instead of int64 too but that's not going to be available everywhere. I'll think about this more.